resources/views/pages/import-files/index.blade.php

```diff
@@ -15,7 +15,9 @@
         @foreach($data as $key => $importFile)
             <tr class="row">
               <td class="col-md-3">
-                <strong>{{$importFile->created_at}}</strong>
+                <a href="/import-files/{{$importFile->id}}">
+                  <strong>{{$importFile->created_at}}</strong>
+                <a href="/import-files/{{$importFile->id}}">
```

Comment:
is this one supposed to be a closing tag?

Reply:
Yes! 🙈Thanks for catching that.